app/assets/stylesheets/finder_frontend.scss

```diff
@@ -66,6 +61,18 @@
   }
 }
 
+.document--top {
+  border: 1px solid $grey-3;
+  padding: govuk-spacing(6);
+  margin-bottom: govuk-spacing(3);
+
+  .document__link {
+    @include bold-24;
+    display: block;
+    margin-top: $gutter-one-third;
```

Comment:
$gutter, only use for the gaps in between grid columns

Try and use GOVUK spacing or responsive spacing https://design-system.service.gov.uk/styles/spacing/